modules/administration/pages/repo-metadata.adoc

```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
+You can manage metadata signing on the command line using the [command]``mgr-sign-metadata-ctl`` command.
 
-After enabling signing metadata, all metadata needs to be re-generated.
-This can be done with a small SQL script:
 
+.Procedure: Enabling Metadata Signing
+. You will need to know the short identifier for the key to use.
+You can list your available keys in short format:
++
 ----
-$> spacewalk-sql -i
-psql (9.6.9)
-Type "help" for help.
-
-susemanager=# insert into rhnRepoRegenQueue (id, CHANNEL_LABEL, REASON, FORCE)
-susemanager-# (select sequence_nextval('rhn_repo_regen_queue_id_seq'), C.label, 'changed signing of metadata', 'Y' from rhnChannel C);
-INSERT 0 40
-susemanager=# \q
-$>
+gpg --keyid-format short --list-keys
 ----
-
-=== Trust the GPG key on all clients
-
-When this feature is enabled, all clients have to trust the new GPG key.
-If the new GPG key is not imported on the client, installation or updating of packages is not possible.
-
-. Export the GPG key and make it availabel in the 'pub/' directory.
 +
-
+. Enable metadata signing with the [command]``mgr-sign-metadata-ctl`` command:
++
 ----
-# gpg --batch --export -a -o <output filename> <KEYID>
-$> gpg --batch --export -a -o /srv/www/htdocs/pub/company.key 607FABDB
+mgr-sign-metadata-ctl enable 4E2C3DD8
+OK. Found key 4E2C3DD8 in keyring.
+DONE. Set key 4E2C3DD8 in /etc/rhn/signing.conf.
+DONE. Enabled metadata signing in /etc/rhn/rhn.conf.
+DONE. Exported key 4E2C3DD8 to /srv/susemanager/salt/gpg/mgr-keyring.gpg.
+DONE. Exported key 4E2C3DD8 to /srv/www/htdocs/pub/mgr-gpg-pub.key.
 ----
-
-. Import the GPG key on all clients
-+
-
-This can be done using a remote command on all clients
+. Restart the metadata services and schedule metadata regeneration to pick up the changes:
 +
-
 ----
-# rpm --import http://<server.domain.top/pub/keyname.key
-$> rpm --import http://suma-refhead-srv.mgr.suse.de/pub/company.key
+mgr-sign-metadata-ctl regen-metadata
 ----
+. OPTIONAL: If you want to regenerate channel metadata with signing enabled, use this command:
 +
+----
+mgr-sign-metadata-ctl regen-metadata
+----
+
+You can also use the [command]``mgr-sign-metadata-ctl`` command to perform other tasks.
+Use [command]``mgr-sign-metadata-ctl --help`` to see the complete list.
+
+Repository metadata signing is a global option.
+When it is enabled, it is enabled on all software channels on the server.
+This means that all clients connected to the server will need to trust the new  GPG key to be able to install or update packages.
 
-[TIP]
-.Tip
-====
-For salt managed systems it might make sense to use a state to trust GPG keys.
-====
+.Procedure: Importing GPG keys on Clients
+
+. On RPM-based client systems, use these remote commands:
++
+----
+rpm --import http://server.example.com/pub/keyname.key
+rpm --import http://server.example.com/pub/company.key
+----
+. For Ubuntu clients, you will need to reassign the channels, which will automatically pick up the new GPG key.
```

Comment:
Reassigning the channels can be done either from the UI (the preferred method) or from the server command line: `salt <ubuntu-minion> state.apply channels`.